controller/userFeedbackController.js

```diff
@@ -0,0 +1,31 @@
+let addUserFeedback = require('../model/addUserFeedback.js')
+
+const userfeedback = async (req, res) => {
+  try {
+    const { name, contact_number, email, experience, comments } = req.body;
+    if (!name) {
+      return res.status(400).send('Name is required');
```

Comment:
Can we standardise all messages sent to be { message: <message> }

Reply:
Resolved